deploy-cloud-run/action.yml

```diff
@@ -45,11 +63,134 @@ runs:
       env:
         IMAGE: ${{ inputs.image }}
         IMAGE_INTERNAL: ${{ inputs.image_internal }}
+    - name: Parse test_url and retrieve the path
+      id: parse_url
+      if: ${{ inputs.test_url != '' }}
+      run: |
+        extract_path() {
+            local url="$1"
+            # Remove scheme (http:// or https://) and domain, keep only the path
+            local path=$(echo "$url" | sed -E 's|https?://[^/]+||')
+            # If path is empty, set to "/"
+            if [[ -z "$path" ]]; then
+                path="/"
+            fi
+            echo "url_path=$path" >> "${GITHUB_OUTPUT}"
+        }
+        extract_path "${{ inputs.test_url }}"
+      shell: bash
+    - name: Get current Cloud Run service revision
+      id: get_revision
+      if: ${{ inputs.automatic_rollback_enabled == 'true' || inputs.blue_green_enabled == 'true' }}
+      shell: bash
+      run: |
+        echo "Getting current cloudrun revision..."
+        REV=$( \
+          gcloud run services describe "${{ inputs.service }}" \
+            --region "${{ inputs.region }}" \
+            --format="value(status.latestReadyRevisionName)" \
+        )
+        echo "Current revision: $REV"
+        echo "revision=$REV" >> "$GITHUB_OUTPUT"
+    - name: Logging - Deploy to Cloud Run
+      run: echo "Deploying to Cloud Run..."
+      if: ${{ inputs.blue_green_enabled == 'false' }}
+      shell: bash
     - name: Deploy to Cloud Run
       uses: google-github-actions/deploy-cloudrun@v2
+      if: ${{ inputs.blue_green_enabled == 'false' }}
+      id: deploy
+      with:
+        service: ${{ inputs.service }}
+        job: ${{ inputs.job }}
+        image: ${{ inputs.image_internal || inputs.image }}
+        region: ${{ inputs.region }}
+        revision_traffic: ${{ inputs.to_latest == 'true' && 'LATEST=100' || '' }}
+        skip_default_labels: true
+    - name: Run HTTP health check
+      continue-on-error: true
+      uses: fjogeleit/http-request-action@v1
+      if: ${{ inputs.blue_green_enabled == 'false' && inputs.test_url != '' }}
+      id: test_live
+      with:
+        url: ${{ inputs.test_url }}
+        method: 'GET'
+        ignoreSsl: true
+        ignoreStatusCodes: ${{ steps.http_responses.outputs.codes }}
```

Comment:
@rot1024 on this phase, if blue-green is disabled and we configure test_url to do live testing to simulates external URL calling

Later I might also add functionality to supply additional parameters such as dynamic method, body, etc. But for now I think using simple GET to specific endpoint that determine the service liveness is sufficient